mappings/net/minecraft/block/entity/BlockEntity.mapping

```diff
@@ -5,6 +5,7 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 	FIELD field_11866 cachedState Lnet/minecraft/class_2680;
 	FIELD field_11867 pos Lnet/minecraft/class_2338;
 	FIELD field_11868 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_19314 hasWarnedAboutInvalidness Z
```

Comment:
```suggestion
	FIELD field_19314 invalid Z
```

Reply:
I disagree with this change, this boolean is only set to true once the warning has been outputted.  If it finds that it is invalid, it triggers the method which checks if this is true, so this isn't if it is invalid, it is if it has already outputted it to prevent console spam.

Reply:
maybe `markedInvalid` then as it's shorter?

Reply:
That works.